apps/core/src/components/NoData.tsx

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Theme } from '../enums';
+import { useTheme } from '../hooks';
+import NoDataImage from '../assets/images/no_data.svg';
+import NoDataDarkImage from '../assets/images/no_data_darkmode.svg';
+
+interface NoDataProps {
+    message: string;
+    displayImage: boolean;
```

Comment:
let's make this prop optional :pray: and add it just when it's neccesary
```suggestion
    displayImage?: boolean;
```